src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
FYI, patternfly is based on bootstrap v3. Accordion component is ready-made for notification-drawer of patternfly. However, in patternfly-webcomponents, I have to create accordion from the scratch and append a few specific css code. 😢